nixos/modules/services/misc/nix-gc.nix

```diff
@@ -61,10 +59,10 @@ in
         '';
       };
 
-      options = mkOption {
+      options = lib.mkOption {
         default = "";
         example = "--max-freed $((64 * 1024**3))";
-        type = types.str;
+        type = lib.types.singleLineStr;
```

Comment:
Doesn't it also make sense to profile multiple options via a multiline string? Even if it doesn't, this might break some peoples config.

Reply:
But how? There's no way back from the `exec`

Reply:
Yeah right.